project/Dependencies.scala

```diff
@@ -689,17 +688,8 @@ object Dependencies {
     akkaPersistenceCassandra
   )
 
-  val `kafka-server` = libraryDependencies ++=
-    // log4j version prior to 1.2.17 required javax.jms, and that artifact could not properly resolved when using maven
-    // without adding a resolver. The problem doesn't appear with sbt because the log4j version brought by both zookeeper
-    // and curator dependencies are evicted to version 1.2.17. Unfortunately, because of how maven resolution works, we
-    // have to explicitly add the desired log4j version we want to use here.
-    // By the way, log4j 1.2.17 and later resolve the javax.jms dependency issue by using geronimo-jms. See
-    // http://stackoverflow.com/questions/4908651/the-following-artifacts-could-not-be-resolved-javax-jmsjmsjar1-1
-    // for more context.
-    log4jModules ++
-    Seq(
-    "org.apache.kafka" %% "kafka" % KafkaVersion exclude("org.slf4j", "slf4j-log4j12"),
```

Comment:
This was a little over eager. Looks like we need to keep the exclusion. I'll add a comment with an explanation.

Reply:
Unfortunately, it's not as simple as just bringing back the exclusion 😦 

With the exclusion, you get this message with `runAll`:

```
SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder".
SLF4J: Defaulting to no-operation (NOP) logger implementation
```

Without it, the broker tests fail.

Reply:
Solved in `excludeLog4jFromKafkaServer` above.